### PR TITLE
Phased rx

### PR DESCRIPTION
**Pull request by Doreban**
Created new experiment that uses pre determined beam directions for beamforming in a full fov experiment to account for edge cases affecting peak power direction. Implemented from Remington's corrected hamming window method in postprocessors.

### CODE REVIEW

**Comment by Doreban** [comment]
See borealis pull request that enabled this experiment: https://github.com/SuperDARNCanada/borealis/pull/432

File: full_fov_phased_rx.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+class FullFOV(ExperimentPrototype):
+    def __init__(self, **kwargs):
+        """
+        kwargs:
+
+        freq: int
+
+        """
+        cpid = 3800
```
Comment (RemingtonRohel) [inline full_fov_phased_rx.py]: This CPID collides with `full_fov.py`. I think the best course of action would be to save this file as `full_fov.py`

**Review by RemingtonRohel — CHANGES_REQUESTED** [review]
Looks great to me, other than a CPID collision that should be resolved.

File: full_fov_phased_rx.py (hunk 1 above)
Comment (Doreban) [reply full_fov_phased_rx.py]: I have moved full_fov_phased_rx to become the new full_fov experiment

**Comment by Doreban** [comment]
Deleting this pull request and branch. Work was moved to branch phased_rx_take_2 and merged into slice_refactor on pull request 11

**Pull request closed without merging**